### ISSUE TITLE
Inconsistent unit failure on away_sites_testing map.

### ISSUE THREAD

**Issue by MistakeNot4892** [NebulaSS13/Nebula#2388]
````
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Rear Compartment Vent Pump #1 with id_tag 638 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #1 with id_tag 637 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #2 with id_tag 639 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #3 with id_tag 644 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #4 with id_tag 647 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Fore Compartment Vent Pump #1 with id_tag 640 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Fore Compartment Vent Pump #2 with id_tag 643 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Fore Compartment Vent Pump #3 with id_tag 646 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Fore Compartment Vent Scrubber #1 with id_tag 641 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Fore Compartment Vent Scrubber #2 with id_tag 642 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Fore Compartment Vent Scrubber #3 with id_tag 645 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Rear Compartment Vent Pump #1 with id_tag 651 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #1 with id_tag 650 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #2 with id_tag 652 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #3 with id_tag 657 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Rear Compartment Vent Scrubber #4 with id_tag 660 did not update the air alarm in area the Desperado Rear Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Fore Compartment Vent Pump #1 with id_tag 653 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Fore Compartment Vent Pump #2 with id_tag 656 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Vent Desperado Fore Compartment Vent Pump #3 with id_tag 659 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Fore Compartment Vent Scrubber #1 with id_tag 654 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Fore Compartment Vent Scrubber #2 with id_tag 655 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: [MAP: Air alarms shall receive updates.]: Scrubber Desperado Fore Compartment Vent Scrubber #3 with id_tag 658 did not update the air alarm in area the Desperado Fore Compartment.
## UNIT_TEST ##: !!! FAILURE !!! [MAP: Air alarms shall receive updates.]: Some areas did not receive updates from all of their atmos devices.
````

**Comment by MistakeNot4892** [comment]
May have been fixed by #2670, reopen if recurs.

**Comment by out-of-phaze** [comment]
Fixed by #2920, I think. Hasn't occurred since, at least.
